src/app/index.js

```diff
@@ -11,6 +12,8 @@ import { InaccessibleContentError } from './errors.js';
 
 const __dirname = path.dirname(new URL(import.meta.url).pathname);
 const SERVICE_DECLARATIONS_PATH = path.resolve(__dirname, '../../', config.get('serviceDeclarationsPath'));
+const NUMBER_OF_TRACK_DOCUMENT_WORKERS = 20;
+const NUMBER_OF_REFILTER_WORKERS = 1;
```

Comment:
Is there any reason for not having a higher parallelization maximum?
If no, it would be great to accelerate that step.
If yes, it seems critical to me to document it.

Reply:
As git is a bottleneck because it chains commands I though it changes nothing but i tried this :
61,8s for 1 workers
52s for 5 workers
51,5s for 10 workers
49,9s for 20 workers
So I will use 20 workers even if the speed up is small.